apps/demo/src/app/cart/components/cart/cart.component.html

```diff
@@ -1,8 +1,10 @@
 <div class="demo-cart">
```

Comment:
Should I remove the wrapper `.demo-cart` element?

Reply:
yea I think that makes sense